conwayconsole/Program.cs

```diff
@@ -50,6 +45,7 @@ public static async Task Main(params string[] args)
                         return;
                     }
 
+                    // Load options
```

Comment:
Weren't you going to extract a method for loading, so the Main() method isn't too bad

Reply:
So I agreed that this could be done under a separate PR.